### PR TITLE
[1LP][RFR] Adding multi_drop down service dialog test

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run

{{pytest: cfme/tests/services/test_dialog_element_in_catalog.py::test_multi_dropdown_dialog_value  }}

### CODE REVIEW

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 1]
```diff
@@ -1003,20 +1003,16 @@ def test_dialog_dropdown_integer_required():
...
-    pass
+    catalog_item, sd, ele_label = generic_catalog_item_with_imported_dialog
+
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalogs, "Order")
+
+    assert "One" and "Two" and "Three" in[
+        option.text for option in view.fields(ele_label).multi_drop.all_options]
+    assert "<None>" not in [
+        option.text for option in view.fields(ele_label).multi_drop.all_options]
```
Comment (valaparthvi) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: This condition will not check what you want to check.
The condition here will only fail if `Three` is not in the list.

You can do this instead - 
```suggestion
    options_list = [option.text for option in view.fields(ele_label).multi_drop.all_options]
    assert all([opt in options_list for opt in ["One", "Two", "Three"]])
    assert "<None>" in options_list
```

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 2]
```diff
@@ -990,10 +990,10 @@ def test_datepicker_in_service_request():
...
 
 
-@pytest.mark.meta(coverage=[1740823])
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_dialog_dropdown_integer_required():
+@pytest.mark.meta(automates=[1740823])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("file_name", ["bz_1740823.yml"], ids=["sample_dialog"],)
+def test_multi_dropdown_dialog_value(appliance, generic_catalog_item_with_imported_dialog):
```
Comment (valaparthvi) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: You forgot to add `file_name` parameter here.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 3]
```diff
@@ -1003,20 +1003,16 @@ def test_dialog_dropdown_integer_required():
         casecomponent: Services
         initialEstimate: 1/16h
         startsin: 5.10
-        testSteps:
```
Comment (valaparthvi) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Why did you remove testSteps and expectedResults?

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 2 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: @valaparthvi here file_name only using for uploading the dialog. in this test we don't need it.

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 3 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: added

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 4]
```diff
@@ -1003,20 +1003,27 @@ def test_dialog_dropdown_integer_required():
...
+            5. "<None>" should not be displayed in the list
     """
-    pass
+    catalog_item, sd, ele_label = generic_catalog_item_with_imported_dialog
+
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalogs, "Order")
+
+    options_list = [option.text for option in view.fields(ele_label).multi_drop.all_options]
+    assert all([opt in options_list for opt in ["One", "Two", "Three"]])
```
Comment (digitronik) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: I would try to use set operation here.
assert set(options_list) == {'One', 'Two', 'Three'}
